openapi/code/named.go

```diff
@@ -95,41 +95,80 @@ func (n *Named) Singular() *Named {
 	return n
 }
 
+func (n *Named) splitASCII() (w []string) {
+	return SplitASCII(n.Name)
+}
+
+// Return the value of cond evaluated at the nearest letter to index i in name.
+// dir determines the direction of search: if true, search forwards, if false,
+// search backwards.
+func search(name string, cond func(rune) bool, dir bool, i int) bool {
```

Comment:
better make these two functions as private methods to Named, not to pollute the global namespace